Gems/PhysX/Code/Source/RigidBody.h

```diff
@@ -24,6 +24,21 @@ namespace PhysX
     class RigidBodyComponent;
     class Shape;
 
+    //! PhysX-specific settings which are not generic enough to be stored in the AzPhysics rigid body configuration.
+    class RigidBodyConfiguration
+    {
+    public:
+        AZ_CLASS_ALLOCATOR(RigidBodyConfiguration, AZ::SystemAllocator, 0);
+        AZ_RTTI(PhysX::RigidBodyConfiguration, "{006A06B1-E2E9-47DF-A49D-C60CAB3727C4}");
+
+        RigidBodyConfiguration() = default;
+
+        static void Reflect(AZ::ReflectContext* context);
+
+        AZ::u8 m_solverPositionIterations = 4; //!< Higher values can improve stability at the cost of performance.
+        AZ::u8 m_solverVelocityIterations = 1; //!< Higher values can improve stability at the cost of performance.
```

Comment:
Is max value of 255 (due to using `AZ::u8`) a limitation of PhysX? If not, why limit it for the user?

Reply:
Yeah, it comes from PhysX